script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -617,7 +617,7 @@ I somewhat awkwardly break eye contact with Arcueid.
 -- アルクェイドの目的が『吸血鬼殺し』だというから、完全に早とちりしていたようだ。
 // identical to first part of Offset 17176
 Because she said that killing vampires was her objective,
- I immediately jumped to what seems to have been the wrong conclusion.
+ I immediately jumped to what seemed to have been the wrong conclusion.
```

Comment:
Seems is correct - it's clear _now_ that it was the wrong conclusion, but at the time it _seemed_ correct